index.html

```diff
@@ -1959,22 +1959,27 @@ <h2>Forms</h2>
 
                         <br><br>
 
-                        <div class="form-check">
-                            <label class="form-check-label">
+                        <div class="form-check form-check-radio">
+                            <label class="form-check-label" for="exampleRadios1">
                                 <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
+                                <div class="form-check-round"></div>
```

Comment:
Consider using an `<i>` instead of a `<div>` as we do with material checkboxes. Otherwise this would lead to a line break when not styled.